### ISSUE TITLE
Be clear that it's not open source

### ISSUE THREAD

**Issue by deepaktrama** [workbenchapp/solana-workbench#130]
The source code for solana-workbench is not present here nor in the releases project.
Please make this clear on the landing page in the interest of not wasting people's time.

**Comment by SvenDowideit** [comment]
huh? this is the repository that has the source - and its the repository that we're building from.

@deepaktrama I'd love to understand what makes you say or think this.

**Comment by SvenDowideit** [comment]
@deepaktrama perhaps this is easier to discus in discord - https://discord.gg/d9WXUhcA

(I wonder if you came from the deprecated pre-open sourcing https://github.com/workbenchapp/solana-workbench-releases repo)

**Comment by SvenDowideit** [comment]
OOOO - @nathanleclaire did we forget to add a LICENSE file? I thought we went MIT!

**Comment by SvenDowideit** [comment]
Note that we kinda did

```
sven@arian:~/src/solana-workbench$ git grep -i license
package.json:  "license": "MIT",
release/app/package.json:  "license": "MIT",
```

**Comment by SvenDowideit** [comment]
@deepaktrama in case this is why you raised this - I just pushed the MIT LICENSE file to the main branch: https://github.com/workbenchapp/solana-workbench/commit/b41430a07613d8591d7c22fc54b730e220d9dcc7

If this is _not_ why you raised this issue, please give us actionable details, as we very much made it open source...

**Comment by nathanleclaire** [comment]
ah, that reminds me https://cryptoworkbench.io/ needs an update

i wonder if we should just delete solana-workbench-releases, or if we link to it from somewhere we overlooked

**Comment by deepaktrama** [comment]
Reasoning -
1. I tried downloading the source code zip files from https://github.com/workbenchapp/solana-workbench-releases/releases and it does not contain any source code but only the readme and image files.
2. I could not find the code for the rust part. Links to the same no longer work, for example. https://github.com/workbenchapp/worknet/blob/7c59295292d3becc3c75b7815153506494d3cf06/programs/worknet/src/lib.rs#L10

Please let me know where I can access the rust code?

**Comment by SvenDowideit** [comment]
so wrt 1. - we open sourced when we released v0.3.0 using _this_ repository, not the `solana-workbench-releases` - see https://github.com/workbenchapp/solana-workbench/releases/tag/v0.3.0 - which is where you'll find the source zip file for the opensourced codebase.

and 2. - what rust code? if you look at the url you pasted, that isn't to the Solana workbench, its to a totally different product&repository. Solana Workbench uses existing on-chain programs to instrument what's happening on chain.

I don't know where you got the link to the internal deep research level codebase - as its not related to Solana Workbench - though it would be fun to know :)

**Comment by SvenDowideit** [comment]
I'm going to close this, as @deepaktrama hasn't replied

but to make sure its totally clear - solana-workbench (the tool in this repo) is fully open source, MIT licensed, and can be built entirely from the code here.
